pol-core/bscript/object.cpp

```diff
@@ -2395,15 +2395,58 @@ bool BFunctionRef::variadic() const
   return variadic_;
 }
 
-BObjectImp* BFunctionRef::call_method_id( const int id, Executor& /*ex*/, bool /*forcebuiltin*/ )
+BObjectImp* BFunctionRef::call_method_id( const int id, Executor& ex, bool /*forcebuiltin*/ )
 {
   switch ( id )
   {
+    // This is only entered if `ins_call_method_id` did _not_ do the call jump.
   case MTH_CALL:
-    return nullptr;  // handled directly
```

Comment:
Isn't the comment still true? At least without knowing where it was, but it's not part of the diff

Reply:
Yes, but I clarified it a bit with the above comment
https://github.com/polserver/polserver/blob/5f7dd8dde48a070acf0e48b667cf1e3bc143b5eb/pol-core/bscript/object.cpp#L2402